menu/views.py

```diff
@@ -14,13 +18,23 @@ class LocationMenuView(generics.RetrieveAPIView):
     search_fields = ['name', 'description']
     ordering_fields = ['name', 'cost', 'volume']
 
+    # @method_decorator(cache_page(30))
```

Comment:
30 sec it's too small period. make variable with this parameter in settings

Reply:
done. change to 5 min